### ISSUE TITLE
Msgs are often reported at the comment block immediately preceding the actual location

### ISSUE THREAD

**Issue by tonyhffong** [tonyhffong/Lint.jl#54]
It's a quirk how julia parser works. It'd be quite difficult to change it within julia itself. For example:

``` julia
# comment. Line 1
function f(x) # Line 2
     1
end
```

"Unused argument x" would be reported on line 1, instead of line 2 where it is declared.
